app/views/measures/grouped/_uk_navigation.html.erb

```diff
@@ -1,3 +1,10 @@
+<!-- UK IMPORT CONTROLS -->
+<% if declarable.import_measures.import_controls.for_country(@search.country).present? %>
```

Comment:
same here,
I guess we'd need to dry out the  `import_measures.import_controls.for_country` part